src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -406,7 +406,8 @@ public static PrintStream makePrintStreamMaybeGzipped(File file) throws IOExcept
     public static File createTempFile(String name, String extension) {
         try {
             final File file = File.createTempFile(name, extension);
-            file.deleteOnExit();
+            // HACK!
+            // file.deleteOnExit();
```

Comment:
Glad it caught your attention, looks like I had forgotten to uncomment that line!